src/scss/custom/layout/_header.scss

```diff
@@ -33,7 +40,10 @@ $component-name: header;
     }
 
     .navbar-brand {
-      padding: 0;
+      .logo {
+        width: auto;
+        max-height: 4rem;
```

Comment:
I've some doubts, should we constraint the height of the logo itself or the navbar? So it adapts to every situations

Would love your opinion @SharakPL @Hlavtox , seems like my integrations skills are getting rusty these days

Reply:
I think a better solution would be to leave img-fluid for the logo and add max-height: 4rem to the `navbar-brand` class as you say @NeOMakinG 
WDYT? @mparvazi

Reply:
fixed height navbar-brand & img-fluid logo

![Screenshot from 2022-08-04 15-43-17](https://user-images.githubusercontent.com/85633460/182833747-ca40c35a-a09e-40ba-8676-64bd48ed2671.png)

Reply:
Dont forget we need to think about CLS, space for the logo must be preserved before it loads, otherwise the page will jump.

Reply:
Something like this with lazy-load script?

```smarty
<img class="logo" 
  src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAAAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==" 
  data-lazy="{$shop.logo_details.src}" 
  alt="{$shop.name}" 
  width="{$shop.logo_details.width}" 
  height="{$shop.logo_details.height}"
>
```

Reply:
I think it's bad practice to give a lazyload above the fold, there may be problems with CLS

Reply:
Yeah that was stupid, forget about that.

Reply:
```smarty
{$maximum_log_height = 75}
{$ratio = 1}
{if $shop.logo_details.height > $maximum_log_height}
    {$ratio = $maximum_log_height / $shop.logo_details.height}
{/if}

<img 
  class="logo img-fluid" 
  src="{$shop.logo_details.src}" 
  width="{$shop.logo_details.width * $ratio}" 
  height="{$shop.logo_details.height * $ratio}"
/>

```

Reply:
It's look fine for me, and the `navbar-brand` with `inline-flex` class just like in your PR